app/Http/Controllers/HomeController.php

```diff
@@ -11,6 +11,7 @@
 use Auth;
 use Illuminate\Support\Facades\Session;
 use App\Http\Requests\SearchRequest;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Namespaces.UseDeclaration.SpaceAfterLastUse</br>**Severity**: error</br>**File**: app/Http/Controllers/HomeController.php</br>**Line**: 13</br>There must be one blank line after the last USE statement; 0 found;